controllers/fenceagentsremediation_controller_test.go

```diff
@@ -312,22 +315,21 @@ func cliCommandsEquality(far *v1alpha1.FenceAgentsRemediation) (bool, error) {
 
 // testPodDeletion tests whether the pod no longer exist for successful FAR CR
 // and consistently check if the pod exist and was not deleted
-func testPodDeletion(podName string, resourceDeletionWasTriggered bool) {
+func testPodDeletion(podName string, wasDeleted bool) {
```

Comment:
The func name with the wasDeleted arg is confusing. I'd prefer 2 functions for clarity. E.g. `verifyPodExists` and `verifyPodDeleted`

Reply:
eventually, the two functions can simply call this one, since the code is basically the same

```
func  verifyPodDeleted(podname string) {
    checkPodIsNotFound(podname, true)
}

func checkPodIsNotFound(podName string, expected bool) {
	podKey := client.ObjectKey{
		Namespace: defaultNamespace,
		Name:      podName,
	}

	ConsistentlyWithOffset(1, func() bool {
		pod := &corev1.Pod{}
		err := k8sClient.Get(context.Background(), podKey, pod)
		return apierrors.IsNotFound(err)
	}, timeoutDeletion, pollInterval).Should(Equal(expected))
}
```

Reply:
>the same

They are close to being the same. One function uses `ConsistentlyWithOffset` and the other uses `EventuallyWithOffset`. But they can both use `ConsistentlyWithOffset`. SGTM

Reply:
oh, I missed that. Maybe that's why I got that problem in my e2e :sweat_smile: